### ISSUE TITLE
Broken integration test suite has me scratching my head

### ISSUE THREAD

**Issue by chartjes** [opencfp/opencfp#1194]
# Steps required to reproduce the problem

1. Check out master branch
2. Run 'make integration'

# Expected Result

* All integration tests pass

# Actual Result

* Integration test suite has failures

I had to alter the phpunit.xml file to disable the recently-merged PHPUnit pretty-print results plugin as it was generating a sprintf() error, no idea what is up there. These are the tests that are currently failing on my laptop. Running PHP 7.2.11

There were 4 failures:

1) OpenCFP\Test\Integration\Http\Action\Talk\EditActionTest::seeEditPageWhenAllowed
Failed asserting that the response body "<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8" />
        <meta http-equiv="refresh" content="0;url=/talk/92" />

        <title>Redirecting to /talk/92</title>
    </head>
    <body>
        Redirecting to <a href="/talk/92">/talk/92</a>.
    </body>
</html>" contains "Dicta debitis accusamus in corrupti et.".
Failed asserting that '<!DOCTYPE html>\n
<html>\n
    <head>\n
        <meta charset="UTF-8" />\n
        <meta http-equiv="refresh" content="0;url=/talk/92" />\n
\n
        <title>Redirecting to /talk/92</title>\n
    </head>\n
    <body>\n
        Redirecting to <a href="/talk/92">/talk/92</a>.\n
    </body>\n
</html>' contains "Dicta debitis accusamus in corrupti et.".

/Users/chartjes/Sites/opencfp/tests/Helper/ResponseHelper.php:60
/Users/chartjes/Sites/opencfp/tests/Integration/Http/Action/Talk/EditActionTest.php:99

2) OpenCFP\Test\Integration\Infrastructure\Auth\SentinelUserTest::checkResetPasswordCodeWorks
Failed asserting that false is true.

/Users/chartjes/Sites/opencfp/tests/Integration/Infrastructure/Auth/SentinelUserTest.php:78

3) OpenCFP\Test\Integration\Infrastructure\Auth\SentinelUserTest::getResetPassWordCodeWorks
Failed asserting that false is true.

/Users/chartjes/Sites/opencfp/tests/Integration/Infrastructure/Auth/SentinelUserTest.php:94

4) OpenCFP\Test\Integration\Infrastructure\Auth\SentinelUserTest::attemptResetPasswordWorks
Failed asserting that false is true.

/Users/chartjes/Sites/opencfp/tests/Integration/Infrastructure/Auth/SentinelUserTest.php:128

Could really use some help to figure out what broke.

**Comment by chrisforrence** [comment]
So that's weird; I'm getting different problems; here's what I got when I ran it using Travis CI:

```
$ vendor/bin/phpunit --testsuite integration
PHPUnit 7.4.0 by Sebastian Bergmann and contributors.
Runtime:       PHP 7.2.10
Configuration: /home/travis/build/opencfp/opencfp/phpunit.xml.dist
........................................................  63 / 176 ( 35%)
..............................................F................ 126 / 176 ( 71%)
...................................F..............              176 / 176 (100%)
Time: 3.93 minutes, Memory: 672.00MB
There were 2 failures:
1) OpenCFP\Test\Integration\Http\Action\Talk\EditActionTest::seeEditPageWhenAllowed
Failed asserting that the response is successful.
Failed asserting that false is true.
/home/travis/build/opencfp/opencfp/tests/Helper/ResponseHelper.php:25
/home/travis/build/opencfp/opencfp/tests/Integration/Http/Action/Talk/EditActionTest.php:101
2) OpenCFP\Test\Integration\Infrastructure\Event\AuthenticationListenerTest::talksRouteWithLogin
Failed asserting that the response status is "200", got "302" instead.
Failed asserting that 302 is identical to 200.
/home/travis/build/opencfp/opencfp/tests/Helper/ResponseHelper.php:40
/home/travis/build/opencfp/opencfp/tests/Integration/Infrastructure/Event/AuthenticationListenerTest.php:56
FAILURES!
Tests: 176, Assertions: 902, Failures: 2.
```

**Comment by chrisforrence** [comment]
OH.

The testing.yml.dist file's CFP ended on October 14th, so CFPs are closed.

**Comment by localheinz** [comment]
Fixed with #1196.
